src/infoview.ts

```diff
@@ -36,26 +38,34 @@ class InfoDocument implements Disposable {
     }
 
     get text() {
-        return this.lines.join('\n');
+        return this.contents;
     }
 
     dispose() {
         for (const s of this.subscriptions) s.dispose();
     }
 
     private render() {
-        this.lines = [];
-        this.renderHeader();
-        this.renderGoal();
-        this.renderMessages();
+        this.contents = this.renderHeader() + this.renderGoal() + this.renderMessages();
+    }
+
+    private stylesheet() {
+        return `#settings { display: block; text-align: right; }
+            h1 {
+                display: block;
+                background-color: blue;
```

Comment:
I think we need a different stylesheet depending on whether we have a light or a dark background:

![info-view-html-bg](https://user-images.githubusercontent.com/313929/27083938-7bebf114-504a-11e7-9be3-8403742991b6.png)